scripts/build-windows-containerd.sh

```diff
@@ -31,41 +31,20 @@ cp bin/ctr.exe /output
 #cp bin/containerd.exe /output # missing CRI plugin, so build from containerd/cri
 cd \$GOPATH
 cd src/github.com/containerd
-git clone https://github.com/containerd/cri.git
+git clone https://github.com/jterry75/cri.git
```

Comment:
Why are we using someone's fork of ContainerD?

Reply:
This the fork the Container Platform team maintains for making the updates required for hyperv.  They are in active conversation with the Containerd team to integrate these changes into the upstream. This was the same process they had for the initial Containerd Support that is now in the 1.4 release.